fe/fe-core/src/main/java/org/apache/doris/datasource/CatalogIf.java

```diff
@@ -153,6 +153,10 @@ default void onClose() {
 
     String getComment();
 
+    default long getLastUpdateTime() {
+        return -1L;
```

Comment:
what the result after TimeUtils.longToTimeString(lastUpdateTime) if set LastUpdateTime to -1 here?